makefile

```diff
@@ -25,3 +25,7 @@ runtests:
 runtests-nocompile:
 	@ cd ./Frontend; make runtests-nocompile -s
 	@ cd ./Backend; make runtests -s
+
+setup:
```

Comment:
What should I do to address this? I don't remember writing this, might have been something from a previous sprint